.travis.yml

```diff
@@ -37,7 +37,6 @@ before_install:
       sudo tee /etc/default/docker > /dev/null
   - sudo service docker restart
   - sleep 5
-  - sudo docker pull centos:centos${OS_VERSION}
```

Comment:
I guess `docker build` will pull automatically if not cached, so this works

Reply:
Ok.